tests/testthat/test-unvalidated-sim_gs_n.R

```diff
@@ -4,14 +4,14 @@
 # See helper-sim_gs_n.R for helper functions
 
 test_that("Test 1: regular logrank test", {
+  # set.seed(2024)
```

Comment:
This was unfortunate timing as all the `sim_gs_n()` tests were commented out. Once #229 is merged, you can rebase on top of that, and then the tests will run.